cmake/modules/CephChecks.cmake

```diff
@@ -155,6 +155,12 @@ __attribute__((__symver__ (\"func@v2\"))) void func_v2() {};
 
 int main() {}"
   HAVE_ATTR_SYMVER)
+  if (NOT HAVE_ATTR_SYMVER)
```

Comment:
@b-ranto could you drop the space after `if`? i pushed a change to your branch to fix it. but it was overwritten by your latest change.

Reply:
Sure, done.